### PR TITLE
Update readme with implementation notes

### PR DESCRIPTION
**Pull request by darryncampbell-pubnub**

### CODE REVIEW

**Review by olivercarson-pubnub — APPROVED** [review]
Changes look good, will merge after PR 17 is merged.

**Comment by darryncampbell-pubnub** [comment]
Merging this since I notice https://github.com/PubNubDevelopers/unity-multiplayer-shooter/pull/17 is merged

**Pull request merged**
